library/classes/POSRef.class.php

```diff
@@ -1,5 +1,24 @@
 <?php
-
+/**
+ * Common script for the encounter form (new and view) scripts.
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Brady Miller <brady@sparmy.com>
+ * @co-author Sherwin Gaddis <sherwingaddis@gmail.com>  
+ * @link    http://www.open-emr.org
+ */
```

Comment:
Thanks for adding the header. Also feel free to place your copyright at the top of the header. The goal is for every script to get a proper header/copyright and it's good to get into the habit of doing this to scripts you modify.